cmd/go-covercheck/root.go

```diff
@@ -271,6 +290,25 @@ func showHistory(cmd *cobra.Command, historyLimit int, cfg *config.Config) error
 	return nil
 }
 
+func deleteHistory(cmd *cobra.Command, deleteRef string, historyLimit int) error {
+	h, err := getHistory(cmd)
+	if err != nil {
+		return fmt.Errorf("failed to load history: %w", err)
+	}
+
+	deleted := h.DeleteByRef(deleteRef)
+	if !deleted {
+		return fmt.Errorf("no history entry found for ref: %s", deleteRef)
+	}
+
+	if err := h.Save(historyLimit); err != nil {
```

Comment:
if saving the file results in no entries at this point, the also prompt for removal of the history file.